src/calendar/tests/unit/CalendarCell.ts

```diff
@@ -91,41 +92,31 @@ registerSuite({
 	},
```

Comment:
For `'Click handler called with correct arguments'()`, could `onClick` be a spy that tests the arguments passed instead of using `clickedDate` and `clickedDisabled`?